README.txt

```diff
@@ -77,8 +77,8 @@ lib [This folder contains 3rd party libraries needed in the project]
 		3 - To compile the test, type in the terminal:
 			make clean; make comp; make all
 		4 - To run the test, type in the terminal "./NAME_OF_THE_COMPILED_FILE". For this specific test you need to type:
-			./SUBNET
-		5 - To check the output of the test, open  "/test/data/subnetsubnet_test_output.txt"
+			./SUBNET_TEST

```

Comment:
This command wouldn’t work if the user doesn’t move bin folder. User would need to move the bin folder and type the above command to run the executable files.

Reply:
It has been fixed with above review. Thanks anyways

Reply:
you're welcome!